CHANGELOG.md

```diff
@@ -1,3 +1,6 @@
+# 0.9.5
+- Added Rocky Linux 8
```

Comment:
This should be a minor release, no?

Reply:
I think there's a point to be made for both. We could view it as a bugfix because centos8 is broken now. If we make it a minor iso4 will not be able to get this change (unless we update the module set file). I'm not sure myself how much weight that should carry. With that in mind, do you still think it would be better to make this a minor? I am fine with both.

Reply:
I don't have a strong preference in either direction.